functions/saveLogEventsToBigQuery.js

```diff
@@ -0,0 +1,59 @@
+
+'use strict';
+
+//-
+// Handling the response. See <a href="https://developers.google.com/bigquery/troubleshooting-errors">
+// Troubleshooting Errors</a> for best practices on how to handle errors.
+//-
+function insertHandler(err, apiResponse) {
+    if (err) {
+      // An API error or partial failure occurred.
+      console.error("Failed to insert table row(s): "+JSON.stringify(err.errors))
+  
+      if (err.name === 'PartialFailureError') {
+        // Some rows failed to insert, while others may have succeeded.
+  
+        // err.errors (object[]):
+        // err.errors[].row (original row object passed to `insert`)
+        // err.errors[].errors[].reason
+        // err.errors[].errors[].message
+      }
+    }
+  }
+  
+  /**
+   * Background Cloud Function to be triggered by Pub/Sub.
+   * This function is exported by index.js, and executed when
+   * the trigger topic receives a message.
+   * 
+   * gcloud functions deploy helloPubSub --runtime nodejs6 --trigger-resource TOPIC_NAME --trigger-event google.pubsub.topic.publish
+   * TOPIC_NAME should be ...
+   *
+   * @param {object} event The Cloud Functions event.
+   * @param {function} callback The callback function.
+   */
+  exports.saveLogEventsToBigQuery = (event, callback) => {
+  
+    const {BigQuery} = require('@google-cloud/bigquery');
+    const projectId = 'data-analysis-pipeline-12387'; 
+    const datasetName = 'logEvents';
+    const tableId = 'logEvents';
```

Comment:
Move out to `config.json` please. See how `TOPIC` is sourced in `logEvents.js` for an example.